lms/djangoapps/course_home_api/outline/serializers.py

```diff
@@ -82,7 +82,7 @@ def get_vertical_icon_class(block):
             video
         """
         children = block.get('children', [])
-        child_classes = {child.get('type') for child in children}
+        child_classes = {value for child in children for value in (child.get('type'), child.get('icon_class'))}
```

Comment:
```suggestion
        child_classes = {
           value 
           for child in children
           for value in (child.get('type'), child.get('icon_class'))
       }
```

for readability I think this will be better

Reply:
@farhaanbukhsh, good idea. Updated in 9a55a41ae95f0ff14befbff7f2edf621de29e807.